client/src/overview/components/Cart.jsx

```diff
@@ -15,11 +16,17 @@ const Cart = ({showCart, cartData, setCart, updateCart}) => {
   };
 
   let countTotal = () => {
-    let total = parseInt(cartData[0].price);
-    for (let i = 1; i < cartData.length; i++) {
-      total += parseInt(cartData[i].price);
+    let priceCount = 0;
+    let itemCount = 0;
+    for (let i = 0; i < cartData.length; i++) {
+      console.log(cartData)
+      console.log(priceCount)
+      priceCount += parseInt(cartData[i].price) * cartData[i].quantity;
+      itemCount += parseInt(cartData[i].quantity)
     }
-    setPrice(total);
+    console.log(priceCount, itemCount)
```

Comment:
remove console logs and its all good to go!

Reply:
thanks!